### ISSUE TITLE
Summary of CNN

### ISSUE THREAD

**Issue by sepidehamiri** [sepidehamiri/personal-research#7]
Three main layers:

## 3. Fully Connected Layers

![image](https://user-images.githubusercontent.com/18999283/34095055-23ae121e-e3e5-11e7-805a-8d7375fafa05.png)

Convolution layers and pooling layers are one among and after them there are some fully connected layers.
### feed forward:
The first feed forward just Multiplication point between input and parameter of neuron.
Calculation of output
Calculation ### loss function
### backpropagation
Calculation gradient of parameter with use chain rule

repeat
